package.json

```diff
@@ -26,8 +28,9 @@
     "validate": "yarn run lint && yarn run typecheck",
     "lint": "eslint .",
     "typecheck": "flow check",
-    "build": "yarn run build:clean && yarn run build:lib && yarn run build:flow && yarn run build:dist",
+    "build": "yarn run build:clean && yarn run build:lib && yarn run build:flow && yarn run build:dist && yarn run build:es",
     "build:clean": "rimraf lib && rimraf dist",
+    "build:es": "cross-env NODE_ENV=production BABEL_ENV=es babel ./src --out-dir ./lib/es",
```

Comment:
`esm` and babel_env is not required here

Reply:
To my knowledge babel uses `BABEL_ENV` to match the `env` section in `.babelrc`. It's possible to rename it here to `BABEL_ENV=esm` (and modify in `.babelrc`) but it won't work if we remove `BABEL_ENV`.

Reply:
I meant rename task to esm and remove BABEL_ENV which is not used.

Reply:
Btw production is not required here too